sdk/data/azappconfig/CHANGELOG.md

```diff
@@ -1,13 +1,20 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.1 (Unreleased)
 
 ### Features Added
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
+### Other Changes
```

Comment:
Did 1.1.0 already release?  Something is wonky in this file.

Reply:
1.1.0 is this release. I added the date for our January Release. (I believe the 16th)